src/azul/project/hca/transformers.py

```diff
@@ -82,6 +86,41 @@ def _cell_suspension_dict(cell_suspension: api.CellSuspension) -> JSON:
     return visitor.merged_cell_suspension
 
 
+def _cell_line_dict(cell_line: api.CellLine) -> JSON:
```

Comment:
Please create ticket about adding meat to cell lines. We should ask @malloryfreeberg what to properties to index.

Reply:
created ticket HumanCellAtlas/metadata-api/issues/51